http/src/main/java/io/micronaut/http/cookie/Cookie.java

```diff
@@ -78,6 +78,23 @@
      */
     long getMaxAge();
 
+    /**
+     * Checks to see if this {@link Cookie} can be sent along cross-site requests.
+     * For more information, please look
+     * <a href="https://tools.ietf.org/html/draft-ietf-httpbis-rfc6265bis-05">here</a>
+     * @return The SameSite attribute of the cookie
+     */
+    @Nullable SameSite getSameSite();
```

Comment:
Maybe prefer `Optional<SameSite>`

Reply:
Also should be a default method that return `Optional.empty()` to avoid breaking API change

Reply:
Done